README.md

```diff
@@ -45,7 +45,6 @@ cmake -DERGOCUB_MODEL_GENERATE_SIMMECHANICS=BOOL:ON -DERGOCUB_MODEL_COPY_TO_SRC=
 make
 (make install)
 ```
-
```

Comment:
You could revert this change on the main README.

Reply:
Do you mean keeping the description of the module and its usage also in the main README?

Reply:
Nope, I've realized that the main README underwent a very minimal (formatting) change that is not wanted/necessary.

Reply:
Ok, I reverted it.